pyproject.toml

```diff
@@ -142,5 +142,6 @@ requires = ["poetry-core>=1.0.0", "poetry-dynamic-versioning"]
 build-backend = "poetry_dynamic_versioning.backend"
 
 [project]
+name = "chia-blockchain"
```

Comment:
Not sure of any ripple effects of this, might be another conflict with poetry like the version.

Reply:
Poetry v2 is supposed to be able to use standard metadata so hopefully whenever that happens we can reconsider https://github.com/Chia-Network/chia-blockchain/pull/18797.

```suggestion
# duplicating until poetry v2, then revisit https://github.com/Chia-Network/chia-blockchain/pull/18797
name = "chia-blockchain"
```

Reply:
taken